core/src/main/java/org/axonframework/config/SagaConfiguration.java

```diff
@@ -53,11 +54,37 @@
     private final Component<AnnotatedSagaManager<S>> sagaManager;
     private final Component<SagaRepository<S>> sagaRepository;
     private final Component<SagaStore<? super S>> sagaStore;
+    /**
+     * @deprecated here only for backward compatibility reasons. {@link EventProcessorRegistry#configureRollbackConfiguration(String,
```

Comment:
I'd  probably rephrase this a little, something like:
```
@deprecated in favor of {@link EventProcessorRegistry#configureRollbackConfiguration(String,Function)}. This function is only kept for backward compatibility.
```

Something similar to the other javadoc around the deprecation.

Reply:
Sounds better, will do it!